src/infra/datasets/postgres/src/repos/postgres_dataset_entry_repository.rs

```diff
@@ -174,6 +174,7 @@ impl DatasetEntryRepository for PostgresDatasetEntryRepository {
                    created_at   as "created_at: _"
             FROM dataset_entries
             WHERE dataset_id = ANY($1)
+            ORDER BY created_at
```

Comment:
Comment for readers:
parity with `SqliteDatasetEntryRepository` ([link](https://github.com/kamu-data/kamu-cli/blob/0df548b9e7e267e4f59986cbd740c6dee9db3a47/src/infra/datasets/sqlite/src/repos/sqlite_dateset_entry_repository.rs#L180))